src/rust/iced-x86/src/info/mod.rs

```diff
@@ -36,6 +36,9 @@ use super::*;
 use alloc::vec::Vec;
 use core::fmt;
 
+extern crate num_traits;
```

Comment:
N.B.: I'm not sure if this is the right place for this -- the CI complained when I put it in the top-level `lib.rs`.

I'm more familiar with Rust >= 2018 than 2015, so forgive me if this is a silly mistake 🙂